mongo/assets/configuration/spec.yaml

```
@@ -0,0 +1,137 @@
+        type: integer
+    - name: ssl_ca_certs
+      description: Path to the ca_certs file.
+      value:
+        type: string
+    - name: additional_metrics
+      description: |
+        By default, the check collects a sample of metrics from MongoDB.
+        This  parameter instructs the check to collect additional metrics on specific topics.
+        Available options are:
+          * `metrics.commands` - Use of database commands
+          * `tcmalloc` -  TCMalloc memory allocator
+          * `top` - Usage statistics for each collection
+          * `collection` - Metrics of the specified collections
+      value:
+        type: array
+        items:
+          type: string
+        example:
+          [metrics.commands, tcmalloc, top, collection]
+    - name: collections
+      description: |
+        Collect metrics on specific collections from the database specified
+        IT Requires `additional_metrics.collection` to be present.
+        Metrics such as `mongodb.collection.count` are collected for each named collection and tagged as follows:
+         * `db:<DB_NAME>` e.g. `db:<DB_NAME>`
+         * `collection:<COLLECTION_NAME>` e.g. `collection:<COLLECTION_NAME>`
+        Each collection generates many metrics, up to 8 + the number of indices on the collection for each collection.
+      value:
+        type: array
+        items:
+          type: string
+        example:
+          - <COLLECTION_NAME>
+    - name: collections_indexes_stats
+      description: |
+        Collect indexes access metrics for every index in every collections in
+        the 'collections' list. This is available starting MongoDB 3.2.
+      value:
+        type: boolean
+        example: false
+    - name: custom_queries
+      description: |
+        Define custom queries to collect custom metrics on your Mongo
+        See https://docs.datadoghq.com/integrations/guide/mongo-custom-query-collection to learn more.
+      value:
+        type: array
+        items:
+          type: object
+          properties:
+            - name: metric_prefix
+              type: string
+            - name: query
+              type: string
+            - name: fields
+              type: object
+              properties:
+                - name: field_name
+                  type: string
+                - name: name
+                  type: string
+                - name: type
+                  type: string
+            - name: tags
+              type: array
+              items:
+                type: string
+        example:
+          - metric_prefix: mongo
+            query: <MONGO_COMMAND_QUERY>
+            fields:
+              - field_name: <FIELD_NAME>
+                name: <METRIC_SUFFIX>
+                type: <METRIC_TYPE>
+            tags:
+              - test:mongodb
+    - template: instances/default
+  - template: logs
+    example:
+    - type: file
+      path: /var/log/mongodb/mongodb.log
+      source: mongo
+      service: mongo
```

This should be provided by the user
```suggestion
      service: <SERVICE>

```